http/typescript/fetch/src/httpCore.ts

```diff
@@ -167,4 +187,53 @@ export class HttpCore implements IHttpCore {
         return url;
     }
 
+    /**
+     * Factory method to create HttpCore instance for url requests not using RequestBuilders
+     * @param coreConfig the configuration used to set authprovider, httpClient or middleware, fetchOptions and more.
+     */
+    public static createCoreForPrimitiveRequests(coreConfig: CoreConfig): HttpCore {
```

Comment:
I'm not following why we need two different methods in this case?

Reply:
The idea to maintain separate factory based on the usage of the core. The goal is achieve something like constructor overloading.

Reply:
I understood that part, what I don't understand is why the initialization would be different from primitive requests to object ones?